src/query/settings/src/settings_default.rs

```diff
@@ -404,7 +404,7 @@ impl DefaultSettings {
                     display_in_show_settings: true,
                 }),
                 ("use_parquet2", DefaultSettingValue {
-                    value: UserSettingValue::UInt64(1),
+                    value: UserSettingValue::UInt64(0),
```

Comment:
One point we need to note is that currently the performance of parquet-rs is not as good as parquet2.

Reply:
It's very closed now and accecptable.
parquet2 is hard to maintain, this is my benchamrk today:
use parquet2=1;
![img_v3_0259_4ed80219-ecce-4c55-9b25-b6b12c82cf5g](https://github.com/datafuselabs/databend/assets/172204/c6c6c6a0-8dfa-49a6-babf-2221e4984e33)

use parquet2=0:
![img_v3_0259_02262d71-5019-49f5-8dfb-179410f20b3g](https://github.com/datafuselabs/databend/assets/172204/7d1ca274-2075-45e0-a3e3-ade2de251c9b)